### PR TITLE
Add Discussion Template for Adding Terms to Glossary

### PR DESCRIPTION
**Pull request by rockleona**
Hi @mattwang44 ,

would you like to have some suggestion toward the template?

### CODE REVIEW

**Comment by rockleona** [comment]
沒有 Preview 可以截圖QQ

File: .github/DISSCUSSION_TEMPLATE/add_term.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+body:
+  - type: markdown
+    attributes:
+      value: |
+        感謝您參與本翻譯計畫 🚀
```
Comment (mattwang44) [inline .github/DISSCUSSION_TEMPLATE/add_term.yml]: 用「你」就好

File: .github/DISSCUSSION_TEMPLATE/add_term.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      options:
+        - label: 我已經確認過 [術語列表](https://github.com/python/python-docs-zh-tw/wiki/%E8%A1%93%E8%AA%9E%E5%88%97%E8%A1%A8) 中沒有相關術語。
+          required: true
+        - label: 我已經確認過 [Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html) 中沒有相關術語。
+          required: true
+
+  - type: input
+    id: term_eng
+    attributes:
+      label: 原文術語
```
Comment (mattwang44) [inline .github/DISSCUSSION_TEMPLATE/add_term.yml]:
```suggestion
      label: 術語原文
```

File: .github/DISSCUSSION_TEMPLATE/add_term.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      description: |
+
+    validations:
+      required: true
+
+  - type: input
+    id: term_zh
+    attributes:
+      label: 翻譯術語
```
Comment (mattwang44) [inline .github/DISSCUSSION_TEMPLATE/add_term.yml]:
```suggestion
      label: 術語翻譯
```

File: .github/DISSCUSSION_TEMPLATE/add_term.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+    validations:
+      required: true
+
+
+  - type: textarea
+    id: reference
+    attributes:
+      label: 參考資料
+      description: 若有其他參考資料，也請麻煩附上，可以加速 Reviewer 的作業流程喔。
```
Comment (mattwang44) [inline .github/DISSCUSSION_TEMPLATE/add_term.yml]: nit
```suggestion
      description: 若有其他參考資料，也請麻煩附上，可以加速 reviewer 的作業流程喔。
```

File: .github/DISSCUSSION_TEMPLATE/add_term.yml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+    validations:
+      required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: 說明
+      description: |
+        請補充說明為什麼想新增詞彙，最好請補充 **原文出處、上下文**。
```
Comment (mattwang44) [inline .github/DISSCUSSION_TEMPLATE/add_term.yml]: nit
```suggestion
        請補充說明原文出處和想新增詞彙的原因。
```

**Review by mattwang44 — COMMENTED** [review]
NP 我覺得可以先 merge & 啟用，使用上遇到問題再回來修正就好！感謝！

**Comment by rockleona** [comment]
修正完畢！

File: .github/DISSCUSSION_TEMPLATE/add_term.yml [hunk 6]
```diff
@@ -0,0 +1,55 @@
+body:
+  - type: markdown
+    attributes:
+      value: |
+        感謝你參與本翻譯計畫 🚀
+
+        謝謝您願意補充術語，讓志工們的翻譯流程更順暢 🙏
+
+        接下來請麻煩依照下列的步驟完成術語補充，我們將在下一次的 meetup 討論您新增的術語，並整理至術語表當中。
```
Comment (mattwang44) [inline .github/DISSCUSSION_TEMPLATE/add_term.yml]: 這邊也要改歐～
```suggestion
        謝謝你願意補充術語，讓志工們的翻譯流程更順暢 🙏

        接下來請麻煩依照下列的步驟完成術語補充，我們將在下一次的 meetup 討論新增的術語，並整理至術語表當中。
```

Comment (rockleona) [reply .github/DISSCUSSION_TEMPLATE/add_term.yml]: 糟糕 我修一下

**Review by mattwang44 — APPROVED** [review]
LGTM!

**Pull request merged**